python_modules/libraries/dagster-sigma/dagster_sigma/assets.py

```diff
@@ -0,0 +1,32 @@
+from typing import cast
+
+from dagster import AssetExecutionContext, AssetsDefinition, AssetSpec, multi_asset
+from dagster._annotations import experimental
+
+
+@experimental
+def build_rebuild_workbook_assets_definition(
```

Comment:
[2] It looks like the term would be materialize in the Sigma ontology. What do you think about `build_materialize_workbook_assets_definition()` instead?

Reply:
Updated